ansible/roles/ansible-horizon-plugin/tasks/uninstall_rhel.yml

```diff
@@ -4,19 +4,30 @@
      update_cache: yes
      name:
         - tvault-horizon-plugin
-        - python-workloadmgrclient
+        - workloadmgrclient
      state: absent
   notify:
      - restart webserver
   when: PYTHON_VERSION=="python2"
   
-- name: uninstall tvault horizon plugin packages when using python3
+- name: uninstall tvault horizon plugin packages when using python3 and ansible_distribution_major_version=="7"
   yum:
      update_cache: yes
      name:
         - python3-tvault-horizon-plugin
-        - python3-workloadmgrclient
+        - python3-workloadmgrclient-el7
      state: absent
   notify:
      - restart webserver
-  when: PYTHON_VERSION=="python3"
+  when: PYTHON_VERSION=="python3" and ansible_distribution_major_version=="7"
+
+- name: uninstall tvault horizon plugin packages when using python3 and ansible_distribution_major_version=="8"
+  yum:
+     update_cache: yes
+     name:
+        - python3-tvault-horizon-plugin
+        - python3-workloadmgrclient-el8
```

Comment:
mentioning wlmclient should not be required. uninstalling horizon-plugin should uninstall all the dependent pkgs.

Reply:
python3-workloadmgrclient-el8 removed for centos-8